interop-tests/src/test/scala/io/grpc/testing/integration/test/AkkaGrpcScalaClientTester.scala

```diff
@@ -26,10 +27,11 @@ import scala.concurrent.{ Await, ExecutionContext, Future }
 import scala.util.Failure
 import scala.util.control.NoStackTrace
 
-class AkkaGrpcScalaClientTester(val settings: Settings)(implicit mat: Materializer, ex: ExecutionContext) extends ClientTester {
+class AkkaGrpcScalaClientTester(val settings: Settings)(implicit mat: Materializer, as: ActorSystem) extends ClientTester {
```

Comment:
nitpick: sys or system would be nicer everywhere... `as` always is a bit weird since `ass` :P but maybe that's just me...